### PR TITLE
Dugk 54 convert to react tabs

### PR DESCRIPTION
**Pull request by DaloonOfDoom**

### CODE REVIEW

**Review by SyntaxXeror — APPROVED** [review]
LGTM 🤠🔫

**Comment by Omarley7** [comment]
```bash
warning " > styled-components@5.3.9" has unmet peer dependency "react-is@>= 16.8.0".
```

Please install missing dependency

**Comment by Omarley7** [comment]
Workbench unittest test failing, likely to be a `<div>` inside a `<p>`

**Comment by Omarley7** [comment]
Looks great few notes, we should take with us in the next PR:
- UI looks very different compared to MUI
- Should really also have a test for the `creatTabdata()` function

File: client/test/unitTests/Components/TabComponent.test.tsx
Comment (Omarley7) [inline client/test/unitTests/Components/TabComponent.test.tsx:1]: Could add a few more test cases:
- What happens if no tabs are provided?
- A test case for the createTabData function

**Review by Omarley7 — CHANGES_REQUESTED** [review]

**Review by Omarley7 — APPROVED** [review]
nice update

**Pull request merged**